lib/collection/src/collection/resharding.rs

```diff
@@ -47,47 +47,28 @@ impl Collection {
 
         shard_holder.start_resharding_unchecked(reshard_key.clone(), replica_set)?;
 
-        // If this peer is responsible for driving the resharding, start the task for it
-        if reshard_key.peer_id == self.this_peer_id {
-            // Stop any already active resharding task to allow starting a new one
-            let mut active_reshard_tasks = self.reshard_tasks.lock().await;
-            let task_result = active_reshard_tasks.stop_task(&reshard_key).await;
-            debug_assert!(task_result.is_none(), "Reshard task already exists");
-
-            let shard_holder = self.shards_holder.clone();
-            let collection_id = self.id.clone();
-            let collection_config = Arc::clone(&self.collection_config);
-            let channel_service = self.channel_service.clone();
-            let progress = Arc::new(Mutex::new(ReshardTaskProgress::new()));
-            let spawned_task = resharding::spawn_resharding_task(
-                shard_holder,
-                progress.clone(),
-                reshard_key.clone(),
-                consensus,
-                collection_id,
-                self.path.clone(),
-                collection_config,
-                channel_service,
-                temp_dir,
-                on_finish,
-                on_error,
-            );
+        Ok(())
+    }
 
-            active_reshard_tasks.add_task(
-                reshard_key,
-                ReshardTaskItem {
-                    task: spawned_task,
-                    started_at: chrono::Utc::now(),
-                    progress,
-                },
-            );
-        }
```

Comment:
Where did this move to?

Reply:
Oops. Temporarily deleted to be able to easily test filtering. I'll revert this. :D